src/main/kotlin/com/amazon/opendistroforelasticsearch/indexstatemanagement/models/ManagedIndex.kt

```diff
@@ -0,0 +1,134 @@
+/*
+ * Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package com.amazon.opendistroforelasticsearch.indexstatemanagement.models
+
+import com.amazon.opendistroforelasticsearch.indexstatemanagement.elasticapi.instant
+import com.amazon.opendistroforelasticsearch.indexstatemanagement.elasticapi.optionalTimeField
+import com.amazon.opendistroforelasticsearch.jobscheduler.spi.ScheduledJobParameter
+import com.amazon.opendistroforelasticsearch.jobscheduler.spi.schedule.Schedule
+import com.amazon.opendistroforelasticsearch.jobscheduler.spi.schedule.ScheduleParser
+import org.apache.logging.log4j.LogManager
+import org.elasticsearch.common.xcontent.ToXContent
+import org.elasticsearch.common.xcontent.XContentBuilder
+import org.elasticsearch.common.xcontent.XContentParser
+import org.elasticsearch.common.xcontent.XContentParser.Token
+import org.elasticsearch.common.xcontent.XContentParserUtils.ensureExpectedToken
+import java.io.IOException
+import java.time.Instant
+
+// TODO: This is an unfinished ManagedIndex data class; just needed for the jobParser
+
+data class ManagedIndex(
```

Comment:
shall we name it as something like `ManagedIndexConfig`?  "Index" has too many meaning, it sounds a little confusing to me.

Reply:
I prefer `ManagedIndex` over `ManagedIndexConfig`; and it follows convention of the other job we currently have in Alerting: `Monitor` instead of `MonitorConfig`.